glib/src/unichar.rs

```diff
@@ -0,0 +1,239 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::{
+    translate::{FromGlib, IntoGlib},
+    UnicodeBreakType, UnicodeScript, UnicodeType,
+};
+
+mod sealed {
+    pub trait Sealed {}
+    impl Sealed for char {}
+}
+
+// rustdoc-stripper-ignore-next
+/// The kind of decomposition to perform
+#[derive(Clone, Copy)]
+pub enum DecompositionKind {
+    // rustdoc-stripper-ignore-next
+    /// Compatbility decomposition
+    Compatibility,
+
+    // rustdoc-stripper-ignore-next
+    /// Canonical decomposition
+    Canonical,
+}
+
+impl DecompositionKind {
+    fn into_bool(self) -> bool {
+        use DecompositionKind::*;
+        match self {
+            Compatibility => true,
+            Canonical => false,
+        }
+    }
+}
+
+// rustdoc-stripper-ignore-next
+/// The result of a single step of the Unicode canonical decomposition algorithm
+pub enum DecompositionResult {
+    // rustdoc-stripper-ignore-next
+    /// The character could not be decomposed further
+    NoDecomposition,
+    // rustdoc-stripper-ignore-next
+    // A 'singleton' decomposition, which means the character was replaced by another
+    Singleton(char),
+    // rustdoc-stripper-ignore-next
+    /// The first character may decompose further, but the second cannot
+    Pair(char, char),
+}
+
+// rustdoc-stripper-ignore-next
+/// This trait provides access to Unicode character classification and manipulations functions
+/// provided by GLib that do not exist in the standard library
+#[doc(alias = "g_unichar")]
+pub trait Unichar: sealed::Sealed + Copy {
+    #[doc(alias = "g_unichar_type")]
```

Comment:
All these functions should be marked as manual in the Gir.toml

Reply:
Will do